src/query.ts

```diff
@@ -170,7 +176,7 @@ class Query {
    *
    * @see {@link https://cloud.google.com/datastore/docs/concepts/queries#datastore-property-filter-nodejs| Datastore Filters}
    *
-   * @param {string} property The field name.
+   * @param {string | NewFilter} propertyOrFilter The field name.
```

Comment:
shouldn't this be EntityFilter now? (side note - how is this compiling?)

Reply:
You are right. This needs to change. This is compiling because this is actually just a documentation comment.

Reply:
Done